utils/orchestrators.ts

```diff
@@ -168,3 +177,52 @@ export const terminateUpdateCgnOrchestratorTask = (
         )
   );
 };
+
+export const trackExceptionAndThrow = (
+  context: IOrchestrationFunctionContext,
+  logPrefix: string
+) => (err: Error | t.Errors, name: string) => {
+  context.log.verbose(
+    err instanceof Error
+      ? `${logPrefix}|ERROR=${err.message}`
+      : `${logPrefix}|ERROR=${readableReport(err)}`
+  );
+  trackException({
+    exception: new Error(`${logPrefix}|ERROR=${String(err)}`),
+    properties: {
+      name
+    }
+  });
+  throw new Error(String(err));
```

Comment:
What's the outcome of `String(err)` when type of `err` is `t.Errors`? Anyway, I'd rather spend a line of code and be more explicit:
```ts
const errMessage = err instanceOf Error ? err.message : readableReport(err);
throw new Error(errMessage);
```

Reply:
(I see you actually did it just above)